### DISCUSSION TITLE
Feature Request: Enable Asset Options Menu on Publish Page

### DISCUSSION THREAD

**Discussion by kmgdevelopment** [craftcms/cms#8623] [Ideas]
I'd like to be able to access the Assets menu options (preview, download, edit, etc.) from within an individual Asset field on the publish page. 

Often times my client needs to download a file or get its source URL but they have to go searching for it in the main Assets panel. And since they don't necessarily know which source is tied to the field they're working on, it can be difficult for them to find the file they need through the per-source search.

(Just a mockup for example purposes, I don't really care what it looks like)

![assets-mockup](https://user-images.githubusercontent.com/1093524/52370040-d00a0a00-2a06-11e9-90c8-ad1ad364fb56.jpg)

**Comment by BenParizek** [comment]
This would be useful for the Form Workflow use case as well. Currently, admins who receive a notification that a form was submitted have to consider adding an attachment to the notification, building a dynamic URL that also links to the asset area to download the asset, or content editors have to take extra steps of navigating over to the asset area, searching for, and downloading each asset from there.

Consider an application process where the form's submission included a PDF for an application or a contest submission that included an image.
